.github/workflows/ci.yml

```
@@ -50,12 +50,14 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Install dependencies
-      run: |
-        sudo apt install verilator libelf-dev
-        pip install -r python-requirements.txt
+    - name: Install Nix
+      uses: cachix/install-nix-action@v24
```

```suggestion
      uses: cachix/install-nix-action@v24
      
    - name: Nix Format Check
      run: nix fmt -- . --check
```

May as well add a quick Nix format check while your at it.